### ISSUE TITLE
Document how to get an invite to the slack account

### ISSUE THREAD

**Issue by kpdecker** [beerjs/chicago#1]
Nice to have the slack account documented, but can't seem to login without an invite from the admin.

**Comment by jden** [comment]
jden here from @beerjs/owners - would there be any interest in a global @beerjs slack?

**Comment by lukewestby** [comment]
@jden definitely! it would be awesome to have channels for each city and also meta channels for general discussion and issues people are having to supplement github

**Comment by befreestudios** [comment]
+1

**Comment by befreestudios** [comment]
@kpdecker I invited you to the Chicago channel
